### ISSUE TITLE
Результат работы программы может отличатся от вызова API

### ISSUE THREAD

**Issue by vessd** [hcodes/yaspeller#110]
```
$ yaspeller --version
5.0.0
```
```
$ yaspeller test.txt --debug
[DEBUG] Get/check config.
[DEBUG] get: /home/sergey/projects/slog_test/test.txt
[DEBUG] post text -> Yandex.Speller API: /home/sergey/projects/slog_test/test.txt
[DEBUG]
    request: 0
    format: plain
    apiFormat: plain
    lang: en,ru
    options: [object Object]
    text: переиспользование
✗ /home/sergey/projects/test.txt 53 ms
-----
Typos: 1
1. переиспользование (1:1, suggest: пере использование)
-----

Checking finished: 0.326 sec.
```
```
$ curl -d "text=переиспользование" https://speller.yandex.net/services/spellservice.json/checkText 2>/dev/null | jq 
[]
```

**Comment by hcodes** [comment]
https://github.com/hcodes/yandex-speller/blob/master/lib/yandex-speller.js#L5
`YaSpeller` ходит на `http`, а в вашем запросе `https`.
Там разные версии API.

**Comment by vessd** [comment]
Не нашёл документацию на `http` версию. Для `https` словарь больше?
Планируется ли переход на `https`?

**Comment by hcodes** [comment]
http://speller.yandex.net/services/spellservice.json/checkText?text=%D0%9F%D0%B5%D1%80%D0%B5%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5

**Comment by vessd** [comment]
Да, я уже попробовал в терминале. В браузере у меня редирект на `https` везде где возможно и я не видел этого варианта.

**Comment by hcodes** [comment]
С чего всё началось: https://github.com/hcodes/yaspeller/issues/84
Можно переключить на `https`, но нужно проверить работу API.

**Comment by vessd** [comment]
Если нужно просто заменить `http` → `https` и `80`→ `443` в lib/post.js, то можно не проверять, всё успешно сломалось.

**Comment by mkpankov** [comment]
Новое API намного лучше. Например, в демо на сайте работает автоматическое склонение и сленговые слова успешно проходят проверку:

```
коммит
коммиты
коммитов
релиз
релизов
```

В наших текстах используется много таких слов и необходимость поддерживать словарь с ручным добавлением туда всех форм и склонений довольно сильно мешает.

Можно ли подробнее исследовать в чём проблема с https?

**Comment by mkpankov** [comment]
Иллюстрация проблемы, которая появилась с недавними выпусками (я думаю с выходом 5.0.0):
https://github.com/ruRust/rustycrate.ru/commit/96fbf62d9d3b86592418dd28c4d13ae404c2a17b
Пришлось в куче слов добавлять множественные формы/склонения/спряжения.
